src/Controls/src/Core/HandlerImpl/Window/Window.Impl.cs

```diff
@@ -26,6 +26,8 @@ public partial class Window : NavigableElement, IWindow, IVisualTreeElement
 
 		internal Toolbar Toolbar { get; }
 
+		public IVisualDiagnosticsLayer VisualDiagnosticsLayer { get; }
```

Comment:
Does this need to be public ?

Reply:
Ideally, yes. It makes it consistent for us to support more toolsets besides Maui Controls, such as Comet. We can make it internal though, remove it from the interface, and then only reference it from Maui Controls `Window`, but if we can make it public that would be ideal.

While we wrote this with the intention of building out UITools tooling (and giving ourselves a great API to work with, if I do say so myself 😉) I think it should be fine to have it be public, should everyone be okay making it so. If anyone would add their own Adorners, once they select something within our Live Visual Tree we can control turning them all off and just showing ours, so we should still be able to control the experience that we need while letting others use it as they see fit.

I'm open to other opinions on this though.